requestmanager/asyncloader/loadattemptqueue/loadattemptqueue.go

```diff
@@ -12,22 +13,25 @@ import (
 // LoadRequest is a request to load the given link for the given request id,
 // with results returned to the given channel
 type LoadRequest struct {
+	p          peer.ID
```

Comment:
Maybe `p` is totally obvious enough to anyone in this code, but consider maybe `peerID`.